src/components/Filter/Filter.css

```diff
@@ -38,8 +38,15 @@
   }
 }
 
-@media all and (min-width: 577px) {
+@media all and (min-width: 580px) {
   .toggle-filter {
     display: none;
   }
-}
+  /* Set the sidebar to fixed position */
+  /* Position the sidebar at the top of the page */
+  .sidebar {
+    order: 2;
+    position: fixed;
+    top: 0;
```

Comment:
Shouldn't the default be top: 0, left: 0? If so, can't we remove this? I'm not 100% sure, so please double check :) Also we can remove the comments.

Reply:
Removed the 'top: 0' - you were right, it appears to be the default. And, removed the comments.